pimd/pim_instance.c

```diff
@@ -178,6 +178,8 @@ static int pim_vrf_enable(struct vrf *vrf)
 
 	zlog_debug("%s: for %s %u", __func__, vrf->name, vrf->vrf_id);
 
+	pim_mroute_socket_enable(pim);
```

Comment:
can you point me to the place that pim_if_create_pimreg uses the mroute socket?  I'm not seeing it

Reply:
Here: pim_if_new->pim_if_add_vif->pim_mroute_add_vif->setsockopt